aiohttp/client.py

```diff
@@ -344,12 +352,22 @@ def request(self,
             for i in skip_auto_headers:
                 skip_headers.add(istr(i))
 
-        if proxy is not None:
+        if proxy is not None and self._session_proxy is None:
```

Comment:
What is the behavior if `proxy is not None and self._session_proxy is not None`?
I think passed `proxy` parameter should take precedence.